packages/parser/src/index.ts

```diff
@@ -28,18 +36,34 @@ export interface IParserContext {
 const WOW_VERSION_LINE_PARSER = /COMBAT_LOG_VERSION,(\d+),ADVANCED_LOG_ENABLED,\d,BUILD_VERSION,([^,]+),(.+)\s*$/;
 
 export class WoWCombatLogParser extends EventEmitter {
+  private _timezone: string;
```

Comment:
might be able to make this a 'public readonly' sine the only place where it's modified is in the constructor?